esmvaltool/cmorizers/data/formatters/datasets/esacci_ozone.ncl

```diff
@@ -87,7 +87,7 @@ begin
     date = cd_calendar(time, 1)
```

Comment:
This script fails after cmorizing the first variable. I think this is due to `YEAR1` and `YEAR2` not being arrays anymore but floats. Replacing YEAR1(vv) by YEAR1 and same for YEAR2 throughout the script should solve the issue.

Reply:
I fixed it a bit, but I think there are some issues with the data and the download instructions. I doubt I have time to fix this, but we may want to merge an create an issue for this

Reply:
I'll take a deeper look at this one tomorrow. The old script works so let's see if this can be fixed or we open an issue as you proposed.